crates/interpreter/src/instructions/host.rs

```diff
@@ -269,28 +231,28 @@ pub fn sstore<WIRE: InterpreterTypes, H: Host + ?Sized>(context: InstructionCont
     let target = context.interpreter.input.target_address();
     let spec_id = context.interpreter.runtime_flag.spec_id();
 
-    // EIP-1706 Disable SSTORE with gasleft lower than call stipend
-    if context
-        .interpreter
-        .runtime_flag
-        .spec_id()
-        .is_enabled_in(ISTANBUL)
-        && context.interpreter.gas.remaining() <= CALL_STIPEND
+    // EIP-2200: Structured Definitions for Net Gas Metering
+    // If gasleft is less than or equal to gas stipend, fail the current call frame with ‘out of gas’ exception.
+    if spec_id.is_enabled_in(ISTANBUL)
+        && context.interpreter.gas.remaining() <= context.interpreter.gas_params.call_stipend()
```

Comment:
This is a reason why static gas for sstore is ommited, this can be simplified in follow up PR